rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -1137,6 +1149,11 @@ void VKGSRender::end()
 			// TODO: Partial memory transfer
 			auto rp = vk::get_render_pass_location(VK_FORMAT_UNDEFINED, ds->info.format, 0);
 			auto render_pass = m_render_passes[rp];
+			if (render_pass == 0)
```

Comment:
I feel all these can be simplified by using a helper function in the anonymous namespace that has the precomputed passes func to avoid duplication all over the place. Declare as inline ofc.